src/netbase.cpp

```diff
@@ -724,3 +724,13 @@ void InterruptSocks5(bool interrupt)
 {
     interruptSocks5Recv = interrupt;
 }
+
+bool SetListenSocketDeferred(const SOCKET& sock)
+{
+    bool ret = false;
+#ifdef TCP_DEFER_ACCEPT
+    static constexpr int set = 1;
```

Comment:
This is the first time I've seen `static constexpr` used correctly in the Bitcoin code, congratulations.